### ISSUE TITLE
Add onLoad Output in ImgComponent

### ISSUE THREAD

**Issue by ohnoitsfraa** [scaleflex/ng-cloudimage-responsive#3]
This is not really an issue but rather something I noticed that was missing from ImgComponent. I copied the code to my repository instead of using the package via NPM so I could add an Output in the original typescript code for the (load) event on the img tag. I needed the EventEmitter so I could determine if the $event.target was in landscape, portrait or panorama.

Maybe this can be added in the future?

What I changed: 

```
...
@Output() imageLoaded: EventEmitter<any> = new EventEmitter<any>();

...
onImageLoad($event) {
    if (!this.isPreview) {
      this.isPreviewLoaded = true;
      this.isLoaded = true;
    } else if (this.isPreviewLoaded) {
      this.isLoaded = true;
    } else {
      this.isPreviewLoaded = true;
    }
    this.imageLoaded.emit($event);
  }
...
```

What I needed the code for:

```
imageLoaded($event) {
    if ($event.target) {
      if ($event.target.width && $event.target.height) {
        $event.target.classList.add(($event.target.width / $event.target.height) >= 1 ? 'landscape' : 'portrait');
        $event.target.classList.add($event.target.width > ($event.target.height * 2) ? 'panorama' : 'non-panorama');
        $event.target.classList.add('loaded');
      }
    }
  }
```

Thanks in advance!

I could also just make a PR if you want?

**Comment by dzmitry-stramavus** [comment]
@ohnoitsfraa hey, 

thanks for opening the issue for this! Yes, that's great.

Yep, go with PR for it. Thanks

**Comment by dzmitry-stramavus** [comment]
#4 ,

available in v1.0.4
